ubuntu/ml/gpu/cuda/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM sarthfrey/ml-gpu-base:latest
+
+# Taken from https://gitlab.com/nvidia/container-images/cuda/-/blob/master/dist/ubuntu16.04/10.1/base/Dockerfile
```

Comment:
had issues with exposing `libcuda.so` with `--gpus all` when this image was built with our internal databricks cuda installation workflow. Copying and using NVIDIA's cuda image installation steps solved that problem.